projects/components/src/utils/test/datagrid/datagrid.wo.ts

```diff
@@ -78,6 +83,26 @@ export class ClrDatagridWidgetObject extends WidgetObject<ClrDatagrid> {
         return Object.keys(this.rows[index].classes);
     }
 
+    /**
+     * Returns the selection type of the grid.
+     */
+    getSelectionType(): GridSelectionType {
+        if (this.findElements('clr-checkbox-wrapper').length !== 0) {
+            return GridSelectionType.Multi;
+        } else if (this.findElements('clr-radio-wrapper').length !== 0) {
+            return GridSelectionType.Single;
+        } else {
+            return GridSelectionType.None;
+        }
+    }
+
+    /**
+     * Clicks the selection icon on the given row.
+     */
+    selectRow(row: number): void {
+        this.click(`#${this.rows[row].attributes['aria-owns']} input`);
```

Comment:
Referring to something by ID  seems weird can't we just use n-th child? I don't remember if I mentioned it earlier.